wandb/sdk/lib/config_util.py

```diff
@@ -120,3 +120,13 @@ def dict_from_config_file(
     for k, v in loaded.items():
         data[k] = v["value"]
     return data
+
+
+def merge_dicts(dest: dict, src: dict) -> dict:
```

Comment:
could you add tests or just document the intended supported cases.
I tried this example: `src={ 'a': [{ 'b': 2 }, { 'd': 4 }]}`, `dst={ 'a': [{ 'c': 3 }, { 'e': 5 }] }` 
and i get the result `{'a': [{'c': 3}, {'e': 5}]}`  
just making sure it shouldn't be `{'a': [{ 'b': 2 , 'c': 3}, {'d': 4, 'e': 5}]}`

Reply:
yes this is expected, we only want to merge on dict keys, not into lists